jsonify.js

```diff
@@ -27,8 +26,7 @@ var lf_processArguments = function(){
 
     while (0 < l_aryArgs.length){
         l_strFlag = l_aryArgs.shift();
-        console.log(l_strFlag);
-        console.log(l_regexType.test(l_strFlag));
+
         if(l_regexDirectory.test(l_strFlag)){
             l_strDirectory = l_aryArgs.shift();
             l_outFile = l_strDirectory + '.json';
```

Comment:
Does `l_strDirectory` contain a filename or a full path inclusive of a filename?